lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -25,8 +25,8 @@ class CyberSourceGateway < Gateway
       self.live_url = 'https://ics2wsa.ic3.com/commerce/1.x/transactionProcessor'
 
       # Schema files can be found here: https://ics2ws.ic3.com/commerce/1.x/transactionProcessor/
-      TEST_XSD_VERSION = '1.159'
-      PRODUCTION_XSD_VERSION = '1.159'
+      TEST_XSD_VERSION = '1.164'
+      PRODUCTION_XSD_VERSION = '1.164'
```

Comment:
The work related to this PR doesn't require an upgrade of the the XSD version, but I saw there was a later version available. Please weigh in if you have a sense of if this should be left as-is, or if upgrading is helpful.

Reply:
Upgrading is good 👍

Reply:
Occurs to me it might be good to get rid of the old XSD, just for tidiness.